mitodl/mitxpro
# Add company to django admin (1-liner)

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #239

#### What's this PR do?
Adds `Company` to Django Admin

#### How should this be manually tested?
Go to Django Admin, there should be a `Companys` link under Ecommerce, you should be able to add and edit `Company` objects
